app/package.json

```
@@ -5,13 +5,14 @@
   "dependencies": {
     "@aragon/api": "^2.0.0-beta.7",
     "@aragon/api-react": "^2.0.0-beta.6",
-    "@aragon/ui": "^0.40.1",
+    "@aragon/ui": "^1.0.0-alpha.23",
```

We might as well use the latest version: :smile: 

```suggestion
    "@aragon/ui": "^1.0.0-alpha.32",
```